nautobot_floor_plan/tables.py

```diff
@@ -43,3 +43,50 @@ class Meta(BaseTable.Meta):
             "tags",
             "actions",
         )
+
+
+class FloorPlanTileTable(BaseTable):
+    # pylint: disable=R0903
+    """Table for list view."""
+
+    floor_plan_tile = tables.Column(empty_values=[])
+    floor_plan = tables.Column(empty_values=[])
```

Comment:
Might make sense to add a column listing and linking to the derived Location as well?

Reply:
Great idea, added location and linked.